packages/ptokens-swap/test/mocks/ptoken-asset.ts

```diff
@@ -13,7 +13,15 @@ export class pTokensProviderMock implements pTokensAssetProvider {
         promi.emit('operationQueued', 'operation-queued-tx-hash')
         promi.emit('operationExecuted', 'operation-executed-tx-hash')
         return resolve('operation-executed-tx-hash')
-      })
+      }),
+    )
+    return promi
+  }
+  makeContractCall(): Promise<string> {
+    const promi = new Promise<string>((resolve) =>
+      setImmediate(() => {
+        return resolve('mock-address')
```

Comment:
Probably not necessary if you return the address in the computePTokenAddress function